context/context.go

```diff
@@ -15,25 +15,35 @@ import (
 	"github.com/pborman/uuid"
 )
 
+const MaxOpVersion = 40000
+const APIVersion = 1
+
 // Any object that is a part of the GlusterD context and needs to be available
 // to other packages should be declared here as exported global variables
 var (
-	MyUUID uuid.UUID
-	Rest   *rest.GDRest
-	TxnFw  *transaction.GDTxnFw
-	Store  *store.GDStore
+	MyUUID          uuid.UUID
+	Rest            *rest.GDRest
+	TxnFw           *transaction.GDTxnFw
+	Store           *store.GDStore
+	GlusterdVersion int
```

Comment:
This should be OpVersion. Not GlusterdVersion. The GlusterD binary version will be a constant and isn't a part of the context.